server/lib/tuist/environment.ex

```diff
@@ -450,6 +446,16 @@ defmodule Tuist.Environment do
     github_oauth_configured?(secrets) and github_app_private_key(secrets) != nil
   end
 
+  def github_app_name do
+    case @env do
+      :dev -> "tuist-dev"
+      :test -> "tuist-test"
+      :can -> "tuist-canary"
+      :stag -> "tuist-staging"
+      :prod -> "tuist"
+    end
+  end
```

Comment:
This doesn't work for on-premise instances. Does it?

Reply:
Yeah, good point, let me put the value for this in secrets instead